WordPress/Classes/Models/Notifications/Notification.swift

```diff
@@ -1,93 +1,95 @@
 import Foundation
+import CocoaLumberjackSwift
 import CoreData
 import WordPressKit
+import WordPressShared
 import FormattableContentKit
 
 // MARK: - Notification Entity
 //
 @objc(Notification)
-class Notification: NSManagedObject {
+public class Notification: NSManagedObject {
     /// Notification Primary Key!
     ///
-    @NSManaged var notificationId: String
+    @NSManaged public var notificationId: String
 
     /// Notification Hash!
     ///
-    @NSManaged var notificationHash: String?
+    @NSManaged public var notificationHash: String?
 
     /// Indicates whether the note was already read, or not
     ///
-    @NSManaged var read: Bool
+    @NSManaged public var read: Bool
 
     /// Associated Resource's Icon, as a plain string
     ///
-    @NSManaged var icon: String?
+    @NSManaged public var icon: String?
 
     /// Noticon resource, associated with this notification
     ///
-    @NSManaged var noticon: String?
+    @NSManaged public var noticon: String?
 
     /// Timestamp as a String
     ///
-    @NSManaged var timestamp: String?
+    @NSManaged public var timestamp: String?
 
     /// Notification Type
     ///
-    @NSManaged var type: String?
+    @NSManaged public var type: String?
 
     /// Associated Resource's URL
     ///
-    @NSManaged var url: String?
+    @NSManaged public var url: String?
 
     /// Plain Title ("1 Like" / Etc)
     ///
-    @NSManaged var title: String?
+    @NSManaged public var title: String?
 
     /// Raw Subject Blocks
     ///
-    @NSManaged var subject: [AnyObject]?
+    @NSManaged public var subject: [AnyObject]?
 
     /// Raw Header Blocks
     ///
-    @NSManaged var header: [AnyObject]?
+    @NSManaged public var header: [AnyObject]?
 
     /// Raw Body Blocks
     ///
-    @NSManaged var body: [AnyObject]?
+    @NSManaged public var body: [AnyObject]?
 
     /// Raw Associated Metadata
     ///
-    @NSManaged var meta: [String: AnyObject]?
+    @NSManaged public var meta: [String: AnyObject]?
 
     /// Timestamp As Date Transient Storage.
     ///
-    fileprivate var cachedTimestampAsDate: Date?
+    public private(set) var cachedTimestampAsDate: Date?
 
-    private let formatter = FormattableContentFormatter()
+    public let formatter = FormattableContentFormatter()
 
     /// Subject Blocks Transient Storage.
     ///
-    fileprivate var cachedSubjectContentGroup: FormattableContentGroup?
+    public var cachedSubjectContentGroup: FormattableContentGroup?
 
     /// Header Blocks Transient Storage.
     ///
-    fileprivate var cachedHeaderContentGroup: FormattableContentGroup?
+    public var cachedHeaderContentGroup: FormattableContentGroup?
```

Comment:
(nit) Let's extract `NotificationsFormattableContentCache` (or similar name) and document that it's for internal purposes. I'd suggest doing in a separate PR (or never).